docs/user_manual/boxes.md

```diff
@@ -0,0 +1,328 @@
+entire plate this way. Instead, these items can be edited to have a Matrix Barcode attribute which represents
+a plate position rather than a physical barcode. Plate position barcodes must follow a specific pattern:
+  * The item's barcode must be made up of two parts: a prefix representing the box + a box position suffix.
+  * The prefix for the box must be unique, but it does not have to match any of the box attributes (name,
+    alias, matrix barcode).
+  * The box position suffix must be in one of two formats: "standard" (A01, F12, where the letter represents
+	  the row and the number represents the column) or "numeric" (001, 002, etc. moving in rows from the top
+left)
+    * Note that for the "standard" format, column numbers less than 10 must have a 0 (zero) added to the
+			beginning of the number. Column 1 would be represented as "01", column 2 would be "02", etc. This
+matches the column numbers on the box Contents diagram.
+    * Note that for the "numeric" format, 001 is the position in the first row, first column, 002 is the
+		  position in the first row, second column, etc. Numbering moves from left to right filling a row, and
+then continues from left to right on the next row, etc.
+So for example, an item with barcode "FIRST-BOX-A01" would correspond to the first row, first column, and an
+item with barcode "FIRST-BOX-B04" would correspond to the second row, fourth column of the same box.
+
+Before filling by barcode pattern on the Edit Box page, the items must first be saved with the barcodes
+representing the box positions, as above.
+Then, go to the Edit Box page by clicking on the box name in the Boxes list page. The Contents section header
+has an Options menu on the far right of the header; hover over it to reveal the menu, and click "Fill by
+Barcode Pattern".
+
+In the Fill by Barcode Pattern dialog box that appears, enter the prefix (from our example above, that would
+be "FIRST-BOX-"). Then select whether the suffix is "Standard" or "Numeric", and click "Fill". The box page
+will refresh and any items with matching barcodes will be placed in their appropriate positions. Note that any
+items that were in the box before the Fill By Barcode Pattern was applied will be removed from the box.
+
+{% include userman-toplink.md %}
+
+{% assign sub = sub | plus: 1 %}
+{% include userman-heading.md section=page.section sub=sub title="Working with Items in a Box" %}
+
+Actions can be performed on all selected items in a box.
+
+First, go to the Edit Box page by clicking on the box name in the Boxes list page. 
+[Select the target positions](#selecting-positions-within-a-box) of the items you wish to act upon.
+In the toolbar at the top of the contents table, several actions are available for the selected items,
+provided that they are all of the same type (sample, library, dilution, pool). Note that not all actions are
+available for all item types.
+
+  * _Edit_: bulk edit the items
```

Comment:
Remove actions list. State instead that the options available are the same as those found on the Samples, Libraries, Dilutions, or Pools list.